content/kotlin/concepts/loops/loops.md

```diff
@@ -0,0 +1,108 @@
+---
+Title: 'Loops'
+Description: 'In kotlin, a loop is a control structure that allows you to repeat a block of code a specified number of times or until a certain condition is met. Loops are useful for automating repetitive tasks and for processing large amounts of data efficiently.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'loop'
+  - 'while'
+  - 'for'
+  - 'do-while'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+In Kotlin, a loop is a control flow statement that allows you to repeat a block of code a certain number of times or until a certain condition is met. Kotlin provides several types of loops, including:
+
+- `for` loop: This loop iterates over a range of values or elements in a collection.
+- `while` loop: This loop repeatedly executes a block of code as long as a certain condition is true.
+- `do-while` loop: This loop is similar to a while loop, but it guarantees that the block of code will be executed at least once before the condition is checked.
+
+## While loop
```

Comment:
You should have the sections in the same order as they're listed in the bulleted list. You can change one or the other, but the order you have in the list seems the most natural: 1. for, 2. while, 3. do-while

Reply:
hmmm alright so I will change the order of this list as the code mentioned below